apps/docs/src/guide/wallets/checking-balances.md

```diff
@@ -1,9 +1,25 @@
-# Checking balances and coins
+# Checking balances
 
-First, one should remember that, with UTXOs, each _coin_ is unique. Each UTXO corresponds to a unique _coin_, and said _coin_ has a corresponding _amount_ (the same way a dollar bill has either 10$ or 5$ face value). So, when you want to query the balance for a given asset ID, you want to query the sum of the amount in each unspent coin. This querying is done very easily with a wallet:
+## Understanding UTXO Model
 
-<<< @/../../../packages/fuel-gauge/src/doc-examples.test.ts#wallet-check-balance{ts:line-numbers}
+In UTXO (Unspent Transaction Output) based systems, each coin is unique, similar to how physical currency bills have different denominations.
 
-If you want to query all the balances (i.e., get the balance for each asset ID in that wallet), you can use the `getBalances` method:
+A UTXO represents a coin with a specific amount, similar to having a $10 or $5 bill. It's crucial to understand this unique feature of UTXOs, as it differs significantly from Ethereum's account-based system.
 
-<<< @/../../../packages/fuel-gauge/src/doc-examples.test.ts#wallet-check-balances{ts:line-numbers}
+In Ethereum, balances are tracked as cumulative totals, similar to a bank account, rather than as distinct 'coins' or 'bills.
```

Comment:
```suggestion
In Ethereum, balances are tracked as cumulative totals, similar to a bank account, rather than as distinct 'coins' or 'bills'.
```
Missing ' at the end.